ci/create_database.py

```diff
@@ -25,13 +26,27 @@ def generate_token(size=12):
         f.write(json.dumps(config))
 
     with open(f'sql-config.cnf', 'w') as f:
-        f.write(f'''
-[client]
+        f.write(f'''[client]
 host={config["host"]}
 user={config["user"]}
 password="{config["password"]}"
 database={config["db"]}
 ''')
+        if 'ssl-ca' in config:
+            f.write(f'ssl-ca={config["ssl-ca"]}\n')
+        if 'ssl-cert' in config:
+            f.write(f'ssl-cert={config["ssl-cert"]}\n')
+        if 'ssl-key' in config:
+            f.write(f'ssl-key={config["ssl-key"]}\n')
+        if 'ssl-mode' in config:
+            f.write(f'ssl-mode={config["ssl-mode"]}\n')
+
+    if os.path.exists('/sql-config/server-ca.pem'):
```

Comment:
Where do these files get used?

Reply:
Your comment doesn't make any sense. If it's expected to be mounted at /sql-config/, then why are you copying the files below to the current directory?

Reply:
Ah I forgot to modify the kubectl line below. We create files in the current directory, including `sql-config.cnf` and `sql-config.json` above, so that we can create a k8s secret.